api/proto/common/v2/common.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package common.v2;
+import "common/common.proto";
+option go_package = "github.com/Layr-Labs/eigenda/api/grpc/common/v2";
+
+
+message BlobHeader {
+	// Blob version
+	uint32 version = 1;
+	repeated uint32 quorum_numbers = 2;
+	common.BlobCommitment commitment = 3;
+	bytes payment_header_hash = 4;
```

Comment:
Should we just include the payment header now so as to minimize api changes in the future?